packages/block-library/src/categories/edit.js

```diff
@@ -117,7 +117,7 @@ class CategoriesEdit extends Component {
 		const selectId = `blocks-category-select-${ instanceId }`;
 		return (
 			<Fragment>
-				<label htmlFor={ selectId } className="screen-reader-text">
+				<label className={ `${ className }__label` } htmlFor={ selectId }>
```

Comment:
This has an unexpected effect when the user provides a custom class name in the Advanced > "Additional CSS Class" field. The `__label` is appended to the custom class name. I think this should just be strictly assigned as the hard-coded `wp-block-categories__label`?

Reply:
That's a good catch, I can make this change